src/server/set_family_test.cc

```diff
@@ -389,6 +389,22 @@ TEST_F(SetFamilyTest, SAddEx) {
   EXPECT_THAT(Run({"saddex", "key", "2", "val"}), IntArg(0));
   AdvanceTime(1000);
   EXPECT_EQ(1, CheckedInt({"sismember", "key", "val"}));
+
+  auto resp = Run({"saddex", "k", "one", "v"});
+  EXPECT_THAT(resp, ErrArg("value is not an integer or out of range"));
+
+  // KEEPTTL support. add field orig with TTL=10
+  EXPECT_THAT(Run({"saddex", "key", "10", "orig"}), IntArg(1));
+  // add fields new and orig with TTL=5 and KEEPTTL=true. orig ttl should be preserved
```

Comment:
the comment says ttl=5 but in this command I see ttl=1

Reply:
Yeah I mixed up the comments, updated now.